### ISSUE TITLE
Can't Run

### ISSUE THREAD

**Issue by spinalgr1990** [ThePotatoRage/rargb-database-browser#1]
Even if I have python installed I cannot run the project. Can you convert it to .exe file ?

**Comment by ThePotatoRage** [comment]
How are you trying to run it? Have you followed my instructions in the README?

**Comment by spinalgr1990** [comment]
Yes .. I have python installed. Plz make a video tutorial or a gif tutorial

**Comment by epmpub** [comment]
> 

first all ,you need to install flow two package thru pip command.
pip install art
pip install tabulate

and use DOS run the follow command:
python3 browser.py

![image](https://github.com/ThePotatoRage/rargb-database-browser/assets/9148510/cffa2408-4e1f-4969-a886-3078d6a9cde0)

**Comment by epmpub** [comment]
where is database of 
![image](https://github.com/ThePotatoRage/rargb-database-browser/assets/9148510/107a67ea-1cb1-4032-9416-5dc3ccf9536c)

where is the database? how can i paste from?

**Comment by spinalgr1990** [comment]
Ι am Windows user ...
